ebpf/xdp_synproxy_kern.c

```diff
@@ -0,0 +1,799 @@
+// SPDX-License-Identifier: LGPL-2.1 OR BSD-2-Clause
```

Comment:
these files look like they are pulled in from somewhere. Where are they from? Did you modify them? Is it necessary to have them in our repo?

Reply:
this is from https://elixir.bootlin.com/linux/latest/source/tools/testing/selftests/bpf/progs/xdp_synproxy_kern.c with modification by removing some code